gamestonk_terminal/cryptocurrency/coinpaprika/coinpaprika_view.py

```diff
@@ -1292,15 +1383,15 @@ def chart(coin_id: str, other_args: List[str]):
         ]
         df = df.set_index(pd.to_datetime(df["Time0"])).drop("Time0", axis=1)
         title = (
-            f"{coin_id}/{ns_parser.vs} from {df.index[0].strftime('%Y/%m/%d')} "
-            f"to {df.index[-1].strftime('%Y/%m/%d')}",
+            f"{coin_id}/{ns_parser.vs} from {df.index[0].strftime('%Y/%m/%d')} to {df.index[-1].strftime('%Y/%m/%d')}",
         )
-
+        df["Volume"] = df["Volume"] / 1000000
```

Comment:
Nothing bad to say here. Just a comment that you may find interesting, Python3 allows you to define a number like this: 1_000_000. Which is great because you can tell straight it's 1M. :)

Reply:
Cool I didn't know about that :) Will be useful for me for sure!